src/python/WMCore/ReqMgr/Web/ReqMgrService.py

```diff
@@ -211,6 +211,27 @@ def __init__(self, app, config, mount):
         identifier = cdict.get("log_reporter", "reqmgr2")
         self.logdb = LogDB(centralurl, identifier)
 
+        # local team cache which will request data from wmstats
+        self.wmstatsurl = config.reqmgr.wmstats_url
```

Comment:
Well, just look at the previous 10 lines and you'll guarantee this config object supports get method. If it does not, then the whole code is doomed to fail :-)
Sorry, I'm all in favor of keeping the code consistent.